requirements.txt

```diff
@@ -1,6 +1,5 @@
 numpy==1.18.1
 pandas==1.0.3
 geopandas==0.6.1
-snakemake-minimal==5.13.0
```

Comment:
Mark, Dominic and I were all able to install snakemake-minimal via conda. Have you tried with a fresh environment? At any rate, maybe instead of changing the requiremnts, you can add note in the README to install snakemake via pip if it doesn't work via conda

Reply:
Yes a note sounds like a good compromise, given that the Python environment should ideally be interchangeable.